doc/devices/ahs_local.rst

```diff
@@ -0,0 +1,105 @@
+The local AHS device
+====================
+
+The local AHS device of the PennyLane-Braket plugin runs Analogue Hamiltonian Simulation on the local Braket SDK. This
+could be either utilizing the processors of your own PC, or those of a `Braket notebook instance <https://docs.aws.amazon.com/braket/latest/developerguide/braket-get-started-create-notebook.html>`_ hosted on AWS.
+
+This device is useful for small-scale simulations in which the time of sending a job to a remote service would add
+an unnecessary overhead. It can also be used for rapid prototyping before running a computation
+on a paid-for remote service.
+
+
+Usage
+~~~~~
+
+After the Braket SDK and the plugin are installed you immediately have access to the `local Braket AHS device <https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html#braket-simulator-ahs-local>`_ in PennyLane.
+
+The local AHS device is not gate-based. Instead, it is compatible with the ``ParametrizedEvolution`` operator from `pulse programming <https://docs.pennylane.ai/en/stable/code/qml_pulse.html>`_ in PennyLane.
+
+To instantiate the local Braket simulator, simply use:
+
+.. code-block:: python
+
+    import pennylane as qml
+    device_local = qml.device("braket.local.ahs", wires=2)
+
+This device can be used with a QNode within PennyLane. It accepts circuits with a single ``ParametrizedEvolution``
+operator based on a ``ParametrizedHamiltonian`` compatible with the simulated hardware. More information about creating
```

Comment:
Do hyper-links work on plugin websites? If not, could do regular links to the respective docs

Reply:
You don't think the link at the end of the paragraph to the pulse section of the PennyLane docs is enough? Making them regular links will change the formatting from the "code" appearance to "link" appearance.

Reply:
If the hyper links `class:'~.ParametrizedHamiltonian'` dont work then yeah I'd prefer regular links over formatting

Reply:
Done!